scripts/pin_the_slow_way.py

```diff
@@ -27,34 +27,42 @@
 
 # The version commented right after the comma is the one available for **Linux** in **defaults**.
 # That version is assumed to be the same in all platforms and the one used to build all packages.
-# The only known incompatibilities at the moment are jpeg 8 vs 9, and icu 54.1 vs 58.*.
+# BE VERY CAREFUL when updating pinnings. Make sure to check https://abi-laboratory.pro/tracker/
+# for any incompatibility between versions.
+# Also be aware that currently the following pinnings are used for both build and run meta.yaml
+# sections. Defining the same pinning for both sections means we need to be more restrictive.
+#
+# For example, we used to define gmp as 'gmp >=5.0.1,<7'. This is wrong as it allows a package
+# to be built against gmp 6.1.0 but run with 5.0.1.
+# As showed by https://abi-laboratory.pro/tracker/timeline/gmp/ binaries built with 6.1.0 are not
+# compatible with 5.0.1 (the other way around would be true).
 pinned = {
           'boost': 'boost 1.64.*',  # 1.61.0
           'boost-cpp': 'boost-cpp 1.64.*',  # NA
           'bzip2': 'bzip2 1.0.*',  # 1.0.6
           'cairo': 'cairo 1.14.*',  # 1.12.18
-          'ffmpeg': 'ffmpeg 2.8.*',  # NA
+          'ffmpeg': 'ffmpeg >=2.8,<2.8.11',  # NA
           'fontconfig': 'fontconfig 2.12.*',  # 2.12.1
-          'freetype': 'freetype 2.7|2.7.*',  # 2.5.5
+          'freetype': 'freetype 2.7',  # 2.5.5
           'geos': 'geos 3.5.1',  # 3.5.0
           'giflib': 'giflib 5.1.*',  # NA
           'glib': 'glib 2.51.*',  # 2.50.2
-          'gmp': 'gmp >=5.0.1,<7', # 6.1.0
-          'harfbuzz': 'harfbuzz 1.3.*',  # 0.9.39
+          'gmp': 'gmp 6.1.*', # 6.1.0
```

Comment:
`gmp` very rarely breaks backward compatibility, therefore I'd suggest doing either, https://github.com/conda-forge/conda-forge.github.io/pull/245 or using conda-build 3's new features for pinning. Pinning to 6.1.* means once 6.2 comes along I'll have to rebuild a lot of packages for no reason.